### DISCUSSION TITLE
Can we get a special repo (like profile readmes) for Codespaces dotfiles?

### DISCUSSION THREAD

**Discussion by MaxLeiter** [community/community#5079] [Codespaces] [answered]
The repo would be cloned and set up in the homedir like normal dotfiles.

**Comment by tanmayeekamath** [comment, marked as answer]
Hey @MaxLeiter, thanks for your feedback:)

Currently, you can personalize your Codespaces across different projects by creating a public dot files repo under your account: https://docs.github.com/en/codespaces/customizing-your-codespace/personalizing-codespaces-for-your-account#dotfiles

Separately to customize the Codespace for a specific project based on your requirements so that all Codespaces created off a repo follow the same config for all developers working on it, you can create a custom devcontainer configuration: https://docs.github.com/en/codespaces/customizing-your-codespace/configuring-codespaces-for-your-project

Can you help clarify the problem you would like to address with your request; and if any of the above functionalities help for the same? :)

**Reply by MaxLeiter** [reply]
I just missed that section of the docs. Thanks a lot!
